memory-bank/02-build-system.md

```diff
@@ -0,0 +1,96 @@
+# Apache Traffic Server - Build System & Configuration
+
+## Build System & Configuration
+
+### CMake Build System (Version 10+)
+```bash
+# Basic build
+cmake -B build
+cmake --build build
+
+# With options
+cmake -B build -DCMAKE_INSTALL_PREFIX=/opt/ats -DBUILD_EXPERIMENTAL_PLUGINS=ON
+
+# Using presets
+cmake --preset dev
```

Comment:
I would change the preset to release and how to list the presets. Most people are users.